app/lib/shared/urls.dart

```diff
@@ -32,42 +32,63 @@ const _trustedTargetHost = [
 final _siteRootUri = Uri.parse('$siteRoot/');
 final _pathRootUri = Uri(path: '/');
 
+enum PkgSubPage {
+  readme,
+  changelog,
+  example,
+  install,
+  versions,
+  score,
+  admin,
+}
```

Comment:
A single documentation comments would be nice :D

Reply:
On top of that, I think `PkgPageTab` may be a better name, renaming...